blocks/newsletter-form/src/save.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Retrieves the translation of text.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-i18n/
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-block-editor/#useBlockProps
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * The save function defines the way in which the different attributes should
+ * be combined into the final markup, which is then serialized by the block
+ * editor into `post_content`.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/#save
+ *
+ * @return {WPElement} Element to render.
+ */
+export default function save(props) {
+
+	const blockProps = useBlockProps.save();
+
+	const { attributes } = props;
+
+	return (
+		<div { ...blockProps }>
+			<div 
+				class="cata-block-streaming-guide-newsletter"
```

Comment:
I feel like the class names in both the Edit and Save components should be updated to match whatever WP produces based on the block name `cata/newsletter`. I think the base would be `wp-block-cata-newsletter`. Maybe there is a way to set that as a variable in here so other classes and be something like ``${base}__bg-img``

Reply:
I added a new attribute to the `block.json` file called `classnameBase` which contains the string `wp-block-cata-newsletter`. This attribute is used to build all the classNames in the edit and save components. Additionally, in both style rules files, I added a sass variable called `$base` equal to the same string and used that to build the class names in the style rules as well. Unfortunately, I still don't know how we can get the block attributes into the style rules in this framework so I defined the same sass var in both style rule files.